config/config.toml

```diff
@@ -5,6 +5,8 @@
 provider_engine = 'pyth_replicator'
 
 product_update_interval_secs = 10
+port = 8000
```

Comment:
health_check_port?

Reply:
`health_check_port` suggests that the port will only be used for health check and if later on we added a new api it will be in another port. Which I think it shouldn't be what we want to do. The `port` will be the port used by this application for health check and possible any other api we might add in the future.

Reply:
Will change this.